docs/pages/management/admin/labels.mdx

```diff
@@ -107,7 +107,7 @@ On the host where you will run your Node, paste the following content into
 version: v3
 teleport:
   # The address of your Auth Service.
-  auth_server: tele.example.com:3025
+  auth_server: tele.example.com:443
```

Comment:
Revert this.

If you want, feel free to mention `proxy_server` first and offer auth as an alternative.